### DISCUSSION TITLE
project organisation

### DISCUSSION THREAD

**Discussion by marcosci** [bigomics/omicsplayground#112] [General]
The current setup is quite cumbersome and needs restructuring.

To use the common workflows around R packages we should turn this repo into an R package and use different repos for all things regarding the playground.

Can we discuss this here?

**Comment by ivokwee** [comment]
I think it's better to discuss this under "Discussions" or on Slack. Let's keep issues to concrete issues, bugs, feature request. Also please be concrete in your statements.

**Comment by marcosci** [comment]
I think turning the playground into an R package is quite concrete, isn't it? 
We can not check, we can not lint, CI/CD is hard because we would have to tailor everything.
